### PR TITLE
Review on my Tribute Page PR

### PR DESCRIPTION
**Pull request by misikirdelelew**
I have added the index page
I have added the CSS Properties

### CODE REVIEW

**Comment by misikirdelelew** [comment]
**Hey! great job so far!**
Changes Requested
Hi Champ!🤗 Great work so far, almost there.
Highlights
I can see that all the needed parts are her, Good job! ✔
Your project is working✔
Great work nesting your files ✔

There are only few steps needed to be addressed before the approval of your project.
Once you implement all the changes you should be ready to submit your project for a re-review 🚀 🚀

Please make sure all the Linters gives no errors.

It's preferred to follow the HTML, CSS best practices & using semantic Tags.

A read me file is important so the users face no problems installing your project in their local machine.

You can document your repo. in a better way, Use the following article to help you [https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md]

Please all the bellow comments before your next submission.

Cheers and happy codding
looking forward for you awesome project 😊

**Review by ishimwezachee — COMMENTED** [review]
Hi @MishDM , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights


## Required Changes ♻️
- [ ]  You should add inline comments to suggest the changes to the student so that the student can see where exact is the problem 
- [ ] Please explain to the student why you are asking those changes 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -25,132 +25,240 @@ <h1 id="title" class="center">Elon Musk</h1>
 					of Neuralink and OpenAI.</p><br>
 			</div>
```
Comment (danielufeli) [inline index.html]: - [ ] Nice comments, kindly add your comments as an inline comment just like the previous reviewer asked, an example will be this one I did.

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @MishDM , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

SoCloseKeeganMichaelKeyGIF


## Highlights
- Nice Comments👌

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danielufeli — APPROVED** [review]
Hi @MishDM  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice Comments Messages✔️

## Optional suggestions

- [OPTIONAL] Kindly always make sure you put your comments under a line or block of code that fits the context the comment is about, you can learn more about GitHub inline comments feature [Here](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**